src/sindri/sindri.py

```diff
@@ -511,13 +513,67 @@ def get_circuit(self, circuit_id: str) -> dict:
 
         return response_json
 
-    def get_proof(self, proof_id: str, include_proof_input: bool = False) -> dict:
+    def get_circuit_smart_contract_verifier(self, circuit_id: str) -> str:
+        """Get the circuit's smart contract verifier code for `circuit_id`.
+
+        Returns:
+
+        - `str`: the smart contract verifier code
+
+        Raises `Sindri.APIError` if:
+
+        - the circuit's circuit type does not support smart contract verifier generation
+        """
+        if self.verbose_level > 0:
+            print(f"Circuit: Get circuit smart contract verifier for circuit_id: {circuit_id}")
```

Comment:
It would be nice to use a logger instead so people can control where the output goes.

Reply:
Nice. i'll make a ticket for this

Reply:
#32